core/primitives/Cargo.toml

```diff
@@ -67,20 +68,27 @@ std = [
 	"rstd/std",
 	"serde",
 	"rustc-hex/std",
-	"twox-hash",
-	"blake2-rfc",
-	"ed25519-dalek",
-	"hex",
+	"twox-hash/std",
+	"blake2-rfc/std",
+	"ed25519-dalek/std",
+	"hex/std",
 	"base58",
 	"substrate-bip39",
 	"tiny-bip39",
 	"serde",
 	"byteorder/std",
 	"rand",
 	"sha2",
-	"schnorrkel",
+	"schnorrkel/std",
 	"regex",
 	"num-traits/std",
 	"externalities",
 	"primitives-storage/std",
 ]
+
+with_crypto = [
```

Comment:
I think I would call it `full_crypto`.

And I would also like to see an comment what the feature does.

Reply:
done